src/index.js

```diff
@@ -0,0 +1 @@
+import './style/styles.css'
```

Comment:
Please run **npx eslint . --fix** before creating a pull request to catch small problems like missing semi-colons. This will reduce the amount of time required to get a PR approved 😃